### DISCUSSION TITLE
日企用交友App 為員工找真愛

### DISCUSSION THREAD

**Discussion by chun1203** [1101-GS4719-C/Python-Community-GS4719-C#330] [閒聊八卦]
文中提到運用AI推測用戶對另一半外貌的喜好，並且充當”軍師“提供意見，增加配對成功率。
沒想到現在連AI都可以成為我們的愛情顧問了XD
[https://udn.com/news/story/6812/6021412?from=udn-catebreaknews_ch2](url)

**Comment by yt7l0** [comment]
感覺比起交友軟體，這個平台相對更可靠安全

**Comment by YuHengChen54** [comment]
這樣就可以找到志同道合的人了欸
